base/log.go

```diff
@@ -11,10 +11,22 @@ import (
 	reporef "github.com/qri-io/qri/repo/ref"
 )
 
+type DatasetLogItem struct {
```

Comment:
I think it's no longer the right place for this struct to be here. Same as the DatasetRef, it's a bit too high up in the chain and forces some other parts of code.

Reply:
I think it's ok here. This struct is specific to the `log` operation, and nothing else should be using it except for higher up functions that also call `base/log`.